srcpkgs/vips/template

```diff
@@ -0,0 +1,72 @@
+# Template file for 'vips'
+pkgname=vips
+version=8.9.2
+revision=1
+build_style=gnu-configure
+build_helper=gir
+configure_args="--enable-introspection=$(vopt_if gir yes no)"
+hostmakedepends="pkgconf"
+makedepends="
+ $(vopt_if hdf5 hdf5-devel)
+ $(vopt_if hdf5 matio-devel)
+ cfitsio-devel
+ expat-devel
+ fftw-devel
+ giflib-devel
+ glib-devel
+ lcms2-devel
+ libexif-devel
+ libgsf-devel
+ libheif-devel
+ libimagequant-devel
+ libjpeg-turbo-devel
+ libopenexr-devel
+ libpng-devel
+ librsvg-devel
+ libwebp-devel
+ orc-devel
+ poppler-glib-devel
+ tiff-devel
+"
+short_desc="Fast image processing with low memory needs"
+maintainer="Isaac Freund <ifreund@ifreund.xyz>"
+license="LGPL-2.1-or-later"
+homepage="https://libvips.github.io/libvips/"
+distfiles="https://github.com/libvips/libvips/releases/download/v${version}/vips-${version}.tar.gz"
+checksum=ae8491b1156cd2eb9cbbaa2fd6caa1dc9ed3ded0b70443d28cd7fea798ab2a27
+python_version=3
+
+build_options="gir hdf5"
+build_options_default="gir"
+desc_option_gir="GObject introspection"
+desc_option_hdf5="HDF5 support"
+
+if [ -z "$CROSS_BUILD" ]; then
+	build_options_default+="hdf5"
```

Comment:
typo here, should have space in front when extending, please fix so we'll see build with options enabled on travis.

Reply:
Fixed!